rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -710,8 +918,7 @@ def load(
 
                 saver.restore(session, checkpoint)
 
-                a_in = train_utils.load_tensor("message_placeholder")
-                b_in = train_utils.load_tensor("label_placeholder")
+                batch_in = train_utils.load_tensor("batch_placeholder")
```

Comment:
is this whole implementation still compatible with existing models trained with the current version?

Reply:
no, there is 1 additional matrix of weights